pages/Frequency/Operations.md

```diff
@@ -0,0 +1,31 @@
+# Operations
+## Method of execution
+DSNP operations are executed on Frequency via on-chain transactions, also known as extrinsic calls.
+An extrinsic is a type of function defined in a Substrate Pallet.  A Pallet is a Substrate runtime module and also a Cargo crate.
```

Comment:
Link to Substrate?

Reply:
Can we include a brief discussion of how "principals" are represented via signatures? For both the uni- and multi-signature types.

Reply:
Not sure what you mean by principals here.

Reply:
We call it that in the Operations list on DSNP -- basically, who is authorizing the transaction.

Reply:
Ah right. Yes. Will add.

Reply:
@wesbiggs Thoughts? <img width="756" alt="image" src="https://user-images.githubusercontent.com/1252199/223137542-eb35b41b-13b1-4edb-8e07-38cd4d822f8b.png">

Reply:
Great, works for me.